master.jinja2

```diff
@@ -40,17 +49,11 @@ context:
 {% endblock settings %}
 {% endblock global_settings %}
 
-
-{% set LAVA_JOB_VISIBILITY = LAVA_JOB_VISIBILITY|default("public") %}
-device_type: {% block device_type %}{% endblock %}
-job_name: {% block job_name %}{% endblock %}
```

Comment:
look how the 'block device_type' and 'block job_name' are used, isn't that enough?
If not it should be changed everywhere.

Reply:
for the current templates that defines {% block device_type %}{% endblock %}, they should still work.
I changed to use the lava_device_type or lava_job_name variables, that would make it a little easier to  specify. we only need to specify the value for the lava_device_type variable in the device template, and we even could just generate that in the ini file to specify that dynamically.

Reply:
fair enough, but if you want to do it, it should be done everywhere.
Also it's not that often probably never we have to change the device_type for a specific board.
Thats why its set in the devices/x15 for instance.

Reply:
anyway, I updated to not change here to avoid affects on the existing files as possible as I can.